### PR TITLE
[FLOC-1036] Update release docs to copy Buildbot tutorial box to Atlas

### PR DESCRIPTION
**Pull request by jongiddy**

### CODE REVIEW

File: docs/gettinginvolved/infrastructure/release-process.rst [hunk 1]
```diff
@@ -324,12 +324,23 @@ Release
...
+
+#. Add the tutorial box to Atlas:
 
    .. note:: Skip this step for a documentation release.
 
-   .. warning:: This step requires ``Vagrant`` and should be performed on your own workstation;
-                **not** on a :doc:`Flocker development machine <vagrant>`.
+   .. prompt:: bash $
+
+      echo http://storage.googleapis.com/clusterhq-vagrant/flocker-tutorial-${VERSION}.box
```
Comment (wallrj) [inline docs/gettinginvolved/infrastructure/release-process.rst]: I know it's verified later when we build the tutorial boxes and run the acceptance tests, but:

`curl --HEAD http://storage.googleapis.com/clusterhq-vagrant/flocker-tutorial-${VERSION}.box`

...might be a useful extra check at this stage to ensure that the box is available at that URL.

```
(test)[vagrant@localhost ~]$ export VERSION=0.3.2

(test)[vagrant@localhost ~]$ curl --HEAD http://storage.googleapis.com/clusterhq-vagrant/flocker-tutorial-${VERSION}.box
HTTP/1.1 200 OK
Expires: Thu, 19 Feb 2015 13:17:00 GMT
Date: Thu, 19 Feb 2015 12:17:00 GMT
Cache-Control: public, max-age=3600
Last-Modified: Wed, 19 Nov 2014 17:11:59 GMT
ETag: "0e71dc22e2b8fa3b874c2928f7bad497"
x-goog-generation: 1416417119309000
x-goog-metageneration: 3
x-goog-stored-content-encoding: identity
x-goog-stored-content-length: 569319802
Content-Type: application/vnd.previewsystems.box
Content-Language: en
x-goog-hash: crc32c=kQeT6Q==
x-goog-hash: md5=DnHcIuK4+juHTCko97rUlw==
x-goog-storage-class: STANDARD
Accept-Ranges: bytes
Vary: Accept-Encoding
Content-Length: 569319802
Server: UploadServer ("Built on Feb 9 2015 19:54:00 (1423540440)")
Alternate-Protocol: 80:quic,p=0.08
```

...as opposed to...

```
(test)[vagrant@localhost ~]$ curl --HEAD http://storage.googleapis.com/clusterhq-vagrant/flocker-tutorial-1.0.0.box
HTTP/1.1 404 Not Found
Content-Type: application/xml; charset=UTF-8
Content-Length: 127
Date: Thu, 19 Feb 2015 12:21:26 GMT
Expires: Thu, 19 Feb 2015 12:21:26 GMT
Cache-Control: private, max-age=0
Server: UploadServer ("Built on Feb 9 2015 19:54:00 (1423540440)")
Alternate-Protocol: 80:quic,p=0.08

```

**Comment by wallrj** [comment]
Thanks @jongiddy 

I didn't test the new `gsutil cp` command, but it looks sane, the `canned ACL` name looks correct (public-read rather than public_read) and I trust that you've tested it yourself.

Please merge.

File: docs/gettinginvolved/infrastructure/release-process.rst [hunk 2]
```diff
@@ -324,12 +324,23 @@ Release
...
+
+   Use the echoed URL as the public link to the Vagrant box, and perform the steps to :ref:`add-vagrant-box-to-atlas`.
```
Comment (wallrj) [inline docs/gettinginvolved/infrastructure/release-process.rst]: Oh, one more thing.
We should automate this using the Atlas API (or something)
 * https://vagrantcloud.com/docs

Create an issue, unless one already exists, and link to it here.

Comment (wallrj) [inline docs/gettinginvolved/infrastructure/release-process.rst]: https://clusterhq.atlassian.net/browse/FLOC-943

File: docs/gettinginvolved/infrastructure/release-process.rst (hunk 1 above)
Comment (adamtheturtle) [inline docs/gettinginvolved/infrastructure/release-process.rst]: This is one of multiple steps we have that Buildbot is doing its job, and we don't check that it is working elsewhere in this process. My preference is to leave this and add it as a follow up issue to any release (hopefully never, next preference a weekly dev release) where it burns us.

**Pull request merged**